src/Infrastructure/PiwikCustomTracking.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+/**
+ * Simple wrapper for tracking methods in Piwik's API
+ *
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class PiwikCustomTracking {
+
+	const SCOPE_VISIT = 'visit';
+	const SCOPE_PAGE = 'page';
+
+	const EVENT_SET_CUSTOM_VARIABLE = 'setCustomVariable';
+	const EVENT_TRACK_GOAL = 'trackGoal';
+
+	private $events = [];
+
+	public function setCustomVariable( int $variableId, string $name, string $value, string $scope ) {
```

Comment:
I intended a simple wrapper for Piwik's tracking API, thus calling the methods the same. I changed it to `trigger*()` to make it clearer.